### PR TITLE
BXMSPROD-1212 Jenkinsfile.prod.nightly UMB message + date suffix format

### PR DESCRIPTION
**Pull request by Ginxo**
https://issues.redhat.com/browse/BXMSPROD-1212

https://github.com/kiegroup/jenkins-pipeline-shared-libraries/pull/144
https://github.com/kiegroup/kogito-pipelines/pull/123

### CODE REVIEW

**Review by mbiarnes — APPROVED** [review]
LGTM

File: vars/pmebuild.groovy [hunk 1]
```diff
@@ -226,4 +227,14 @@ def getDefaultBranch(Map<String, Object> projectConfig, String currentBranch) {
     return projectConfig != null && projectConfig['scmRevision'] ? projectConfig['scmRevision'] : currentBranch
 }
 
+/**
+ * Saves build project ok in ALREADY_BUILT_PROJECTS env var
+ *
+ * @param project the project name (this should match with the builds.project from the file)
+ */
+def saveBuildProjectOk(String project){
+    env.ALREADY_BUILT_PROJECTS = env.ALREADY_BUILT_PROJECTS != null ?: ""
```
Comment (radtriste) [inline vars/pmebuild.groovy:236]: I think you don't need here the `! =null`

**Review by radtriste — APPROVED** [review]
Minor comment

File: vars/pmebuild.groovy (hunk 1 above)
Comment (Ginxo) [reply vars/pmebuild.groovy]: improved. Thanks @radtriste

**Comment by Ginxo** [comment]
@mareknovotny @radtriste @jstastny-cz can you please review/merge?

**Review by mareknovotny — APPROVED** [review]
lgtm

**Review by radtriste — APPROVED** [review]

**Review by jstastny-cz — APPROVED** [review]

**Pull request merged**